core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -586,62 +586,6 @@ LogicalAggregate(group=[{7, 10}])
   LogicalJoin(condition=[=($7, $9)], joinType=[left])
     LogicalTableScan(table=[[CATALOG, SALES, EMP]])
     LogicalTableScan(table=[[CATALOG, SALES, DEPT]])
-]]>
```

Comment:
We can clearly see that such a check was long due

Reply:
Yes, I actually overlooked this problem. @NobiGo  spotted it when reviewing my code, so I tried to fix it.